pages/password/reset/index.tsx

```diff
@@ -0,0 +1,7 @@
+import ResetPassword from '../../../components/views/auth/ResetPassword'
```

Comment:
Not sure why you deleted my page with the token. Remove this page, move this code into that page after it was restored. We need the token to be passed in via the page url for authentication purposes on both the UI and server